code/modules/food_and_drinks/recipes/food_mixtures.dm

```diff
@@ -140,6 +140,11 @@
 	results = list(/datum/reagent/consumable/nutriment/peptides = 0.5)
 	required_reagents = list(/datum/reagent/consumable/nutriment/ = 0.5)
 	required_catalysts = list(/datum/reagent/medicine/metafactor = 0.5)
+	
+/datum/chemical_reaction/nutriconversion2
```

Comment:
Could you use a better name than this one? What makes it different from the other reaction?
Ideally on `snake_case` instead.

Reply:
I have no clue what I'm doing but okay.